FullStorySegmentMiddleware/Classes/FullStoryMiddleware.m

```diff
@@ -0,0 +1,142 @@
+//
+//  FullStoryMiddleware.m
+//  ios-shoppe-demo
+//
+//  Created on 5/15/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <Analytics/SEGMiddleware.h>
+#import "FullStoryMiddleware.h"
+#import <FullStory/FullStory.h>
+
+@implementation FullStoryMiddleware
+
+- (id)initWithWhitelistEvents:(NSArray<NSString *> *) whitelistEvents {
```

Comment:
Please go ahead and add HeaderDoc comments to these methods. 

I found this examples:

```/*!
 * @discussion A really simple way to calculate the sum of two numbers.
 * @param firstNumber An NSInteger to be used in the summation of two numbers
 * @param secondNumber The second half of the equation. 
 * @return The sum of the two numbers passed in.
 */
+ (NSInteger)addNumber:(NSInteger)firstNumber toNumber:(NSInteger)secondNumber;
```

here: https://www.raywenderlich.com/2397-documenting-in-xcode-with-headerdoc-tutorial

double-check with @HaroldFS too